AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -304,6 +304,8 @@ texture-packs.com#%#//scriptlet("abort-on-property-read", "mdpDeBlockerDestroyer
 ||smedata.sk/monitor/bundles/smeblockmanagerapi/build/js/gfcc.js
 ||s.yimg.com/*/bundle_basic_mail_affiliate_partner_$domain=mail.yahoo.com
 oneman.gr#%#AG_abortInlineScript(/-10000px/, 'window.atob');
+@@||evades.io/ads*.js
+evades.io#%#//scriptlet("set-constant", "window.adBlockerDetected", "false")
```

Comment:
I think that in case like this, it is not necessary to use `window.adBlockerDetected`, just `adBlockerDetected` should be fine.

By the way, with AdGuard for Windows 7.5 nightly 8, using `window.someProperty` seems to not work correctly, so could you please change it to:
`evades.io#%#//scriptlet("set-constant", "adBlockerDetected", "false")`

Reply:
ok